src/lib/FlightTasks/tasks/AutoMapper/FlightTaskAutoMapper.cpp

```diff
@@ -62,6 +62,12 @@ bool FlightTaskAutoMapper::update()
 		_reset();
 	}
 
+	// check if an external yaw handler is active and if yes, let it compute the yaw setpoints
+	if (_ext_yaw_handler != nullptr && _ext_yaw_handler->is_active()) {
+		_yaw_setpoint = _yaw;
+		_yawspeed_setpoint = _ext_yaw_handler->get_weathervane_yawrate();
+	}
+
```

Comment:
would it not better fit to [here](https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Auto/FlightTaskAuto.cpp#L223-L277)?